apps/estuary/test/unit/estuary/services/event_retrieval_service_test.exs

```diff
@@ -3,26 +3,28 @@ defmodule Estuary.Services.EventRetrievalServiceTest do
   use Placebo
 
   alias Estuary.Services.EventRetrievalService
-  alias Estuary.Query.Select
+  alias Estuary.Query.Helper.PrestigeHelper
 
   @tag capture_log: true
   test "should return the events from the table" do
-    expected_events = [
-      %{
-        "author" => "Author-2020-01-21 23:29:20.171519Z",
-        "create_ts" => 1_579_649_360,
-        "data" => "Data-2020-01-21 23:29:20.171538Z",
-        "type" => "Type-2020-01-21 23:29:20.171543Z"
-      },
-      %{
-        "author" => "Author-2020-01-21 23:25:52.522084Z",
-        "create_ts" => 1_579_649_152,
-        "data" => "Data-2020-01-21 23:25:52.522107Z",
-        "type" => "Type-2020-01-21 23:25:52.522111Z"
-      }
-    ]
+    expected_events =
+      {:ok,
+       [
+         %{
+           "author" => "Author-2020-01-21 23:29:20.171519Z",
+           "create_ts" => 1_579_649_360,
+           "data" => "Data-2020-01-21 23:29:20.171538Z",
+           "type" => "Type-2020-01-21 23:29:20.171543Z"
+         },
+         %{
+           "author" => "Author-2020-01-21 23:25:52.522084Z",
+           "create_ts" => 1_579_649_152,
+           "data" => "Data-2020-01-21 23:25:52.522107Z",
+           "type" => "Type-2020-01-21 23:25:52.522111Z"
+         }
+       ]}
 
-    allow(Select.select_table(any()), return: {:ok, expected_events})
+    allow(PrestigeHelper.execute_query(any()), return: expected_events)
     assert expected_events = EventRetrievalService.get_all()
```

Comment:
This test doesn't seem to add much value.  You're basically just testing that the value that you have mocked is returned.  Maybe this code would be covered better by an integration test?